public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -0,0 +1,466 @@
+    surveyVersions: Array<any>;
+    showAnswer: {};
+    questions: Array<any>;
+    loading: boolean;
+    showStatement: boolean;
+}
+
+export class SurveyViewReactComponent extends React.Component<Props, State> {
+    search = _.debounce((val) => this.getSurvey(), 1000);
+    searchCount = _.debounce((val) => this.fetchSurvey(this.props.domainId), 1000);
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isCopeSurvey: false,
+            survey: null,
+            surveyPdfUrl: '',
+            searchWord: this.props.searchWord,
+            extraQuestionConceptIds: [],
+            showAnswer: {},
+            questions: [],
+            loading: true,
+            surveyVersions: [],
+            showStatement: false,
+        };
+        this.changeResults = this.changeResults.bind(this);
+        this.closePopUp = this.closePopUp.bind(this);
+        this.showCopeStatement = this.showCopeStatement.bind(this);
+    }
+
+    componentDidMount() {
+        this.fetchSurvey(this.props.domainId);
+    }
+
+    changeResults(val) {
+        this.fetchSurvey(val);
+    }
+
+    closePopUp() {
+        this.setState({showStatement: false});
+    }
+
+    showCopeStatement() {
+        this.setState({showStatement: true});
+    }
+
+    fetchSurvey(domain) {
+        const {domainId} = this.props;
+        const {searchWord} = this.state;
+        let fetchDomain = domainId;
+        if (domain && domainId !== domain) {
+            fetchDomain = domain;
+        }
+        api.getDomainTotals(searchWord, 1, 1).then(
+                (data: any) => {
+                  data.surveyModules.forEach(survey => {
+                    const surveyRoute = survey.name.split(' ').join('-').toLowerCase();
+                    if (surveyRoute.indexOf('(cope)') > -1) {
+                        if (fetchDomain && surveyRoute.indexOf(fetchDomain) > -1) {
+                            localStorage.setItem('surveyModule', JSON.stringify(survey));
+                            this.setSurvey(JSON.stringify(survey));
+                        }
+                    } else {
+                        if (surveyRoute === fetchDomain) {
+                            localStorage.setItem('surveyModule', JSON.stringify(survey));
+                            this.setSurvey(JSON.stringify(survey));
+                        }
+                    }
+                  });
+                });
+    }
+
+    setSurvey(surveyObj) {
+        if (surveyObj) {
+            const survey = JSON.parse(surveyObj);
+            const surveyConceptId = survey.conceptId;
+            let surveyPdfUrl = '';
+            if (surveyConceptId === 43528895) {
+                surveyPdfUrl = '/assets/surveys/' +
+                'Health Care Access Utilization'.split(' ').join('_') + '.pdf';
+            } else {
+                surveyPdfUrl = '/assets/surveys/' + survey.name.split(' ').join('_') + '.pdf';
+            }
```

Comment:
Nit: Can be simplified to one line:
```suggestion
            const surveyPdfUrl = surveyConceptId === 43528895) ?
              surveyPdfUrl = '/assets/surveys/' + 'Health Care Access Utilization'.split(' ').join('_') + '.pdf' :
              surveyPdfUrl = '/assets/surveys/' + survey.name.split(' ').join('_') + '.pdf';
```

Reply:
Changed this thanks.